tests/late_joiners.py

```diff
@@ -36,117 +35,92 @@ def timeout_handler(node, suspend, election_timeout):
         node.resume()
 
 
-def find_primary(network):
-    # track if there is a new primary
-    term_info = {}
+def update_term_info(network, term_info):
     try:
         cur_primary, cur_term = network.find_primary()
         term_info[cur_term] = cur_primary.node_id
     except TimeoutError:
         LOG.info("Trying to access a suspended network")
-    return term_info
 
 
-def get_node_local_commit(node):
-    with node.node_client() as c:
-        r = c.get("getLocalCommit")
-        return r.commit, r.global_commit
+@reqs.description("Running transactions against logging app")
+@reqs.supports_methods("LOG_record", "LOG_record_pub", "LOG_get", "LOG_get_pub")
+@reqs.at_least_n_nodes(3)
```

Comment:
I think this test would work with any number of nodes? Also, since this test is generic, perhaps it could be moved to `logging_app.py`?